riak_test/yokozuna_essential.erl

```diff
@@ -79,6 +79,7 @@ confirm() ->
             verify_deletes(Cluster2, ?INDEX, ?NUM_KEYS, KeysDeleted),
             ok = test_escaped_key(Cluster2),
             verify_unique_id(Cluster2, PBConns),
+            %% verify_delete_index(Cluster2, PBConns),
```

Comment:
Yes - this test was added to validate that we stop the appropriate solrq pairs when the index for which they were started is removed.